src/main/java/com/ing/data/cassandra/jdbc/CassandraResultSet.java

```diff
@@ -373,6 +375,21 @@ public InputStream getAsciiStream(final int columnIndex) throws SQLException {
         }
     }
 
+    public Array getArray(int columnIndex) throws SQLException {
+        checkIndex(columnIndex);
+        Object o = currentRow.getObject(columnIndex - 1);
+        if (!(o instanceof List)) return null;
```

Comment:
To be compliant with the Checkstyle rules, the `if` block must always use braces.